launcher/test-framework/org.jboss.tools.common.launcher.reddeer/src/org/jboss/tools/common/launcher/reddeer/wizards/NewLauncherProjectWizardPage.java

```diff
@@ -0,0 +1,195 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributor:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.common.launcher.reddeer.wizards;
+
+import org.eclipse.jface.wizard.WizardPage;
+import org.eclipse.reddeer.common.logging.Logger;
+import org.eclipse.reddeer.common.wait.TimePeriod;
+import org.eclipse.reddeer.common.wait.WaitWhile;
+import org.eclipse.reddeer.swt.api.Button;
+import org.eclipse.reddeer.swt.impl.button.CheckBox;
+import org.eclipse.reddeer.swt.impl.button.PushButton;
+import org.eclipse.reddeer.swt.impl.combo.DefaultCombo;
+import org.eclipse.reddeer.swt.impl.text.LabeledText;
+import org.eclipse.reddeer.workbench.core.condition.JobIsRunning;
+import org.eclipse.swt.widgets.Composite;
+
+
+/**
+ * 
+ * @author zcervink
+ *
+ */
+public class NewLauncherProjectWizardPage extends WizardPage {
+	
+	@Override
+	public void createControl(Composite arg0) {
+		// TODO Auto-generated method stub
+		
+	}
+	
+	public NewLauncherProjectWizardPage(String pageName) {
+		super(pageName);
+		// TODO Auto-generated constructor stub
+	}
+
+	protected final static Logger log = Logger
+			.getLogger(NewLauncherProjectWizardPage.class);
+
+	/**
+	 * Finishes this wizard without possibility to check
+	 * NewProjectExamplesReadyPage.
+	 */
+	public void finish() {	
+		Button button = new PushButton("Finish");
+		button.click();
+		new WaitWhile(new JobIsRunning(), TimePeriod.getCustom(1800));
+	}
+	
+	/**
+	 * Sets target mission
+	 * @param targetMissionName
+	 */
+	public void setTargetMission(String targetMissionName) {
+		new DefaultCombo(0).setSelection(targetMissionName);
+	}
+
+	/**
+	 * Sets nth target mission.
+	 * @param index
+	 */
+	public void setTargetMission(int index) {
+		new DefaultCombo(0).setSelection(index);
+	}
+	
+	/**
+	 * Sets target runtime.
+	 * @param targetRuntimeName
+	 */
+	public void setTargetRuntime(String targetRuntimeName) {
+		new DefaultCombo(1).setSelection(targetRuntimeName);
+	}
+	
+	/**
+	 * Sets nth target runtime.
+	 * @param index
+	 */
+	public void setTargetRuntime(int index) {
+		new DefaultCombo(1).setSelection(index);
+	}
+
+	/**
+	 * Toggles project path checkbox
+	 * @param newCheckboxState
+	 */
+	public void toggleTheCheckBox(boolean newCheckboxState){
```

Comment:
@zcervink what checkbox? please rename this method to something more describing - checkbox is very general, new name could be just something like "useDefaultLocation"

Reply:
renamed.